Cargo.lock

```diff
@@ -2000,15 +1976,15 @@ dependencies = [
 
 [[package]]
 name = "rustversion"
-version = "1.0.5"
+version = "1.0.6"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "61b3909d758bb75c79f23d4736fac9433868679d3ad2ea7a61e3c25cfda9a088"
+checksum = "f2cc38e8fa666e2de3c4aba7edeb5ffc5246c1c2ed0e3d17e560aeeba736b23f"
 
 [[package]]
 name = "ryu"
-version = "1.0.5"
+version = "1.0.9"
```

Comment:
Hmm, did you do a `cargo update` here?

I think instead of bumping everything, what we should do is split out the "use ostree-ext from git main" into a separate PR, then just `cargo update -p ostree-ext` there.

As is this will conflict with what dependabot is doing.

Reply:
Also part of the above. I called `cargo update`, not `cargo update -p ostree-ext`. That might be the issue.